package.json

```diff
@@ -41,15 +41,19 @@
     "recursive-readdir": "^2.2.2",
     "sc-istanbul": "^0.4.5",
     "shelljs": "^0.8.3",
-    "web3": "^1.3.0"
+    "web3": "1.2.9"
```

Comment:
Why is this set to 1.2.9?

Reply:
@junderw Good catch! (And thanks for reviewing.)

Initially set this at 1.2.9 because later versions crashed Hardhat when fetching the initial accounts and client version info. Think there is a bug (or unresolved design issue) in Web3 1.3.0's implementation of the EIP1193 Provider. 

However, in a subsequent revision began making those calls with a thin wrapper in nomiclabsUtils. Web3 could be completely removed from the project if all the plugins use this...

https://github.com/sc-forks/solidity-coverage/blob/2cf65306124e594343afc80310398cf09e5b0c23/plugins/resources/nomiclabs.utils.js#L159-L165

Reply:
ok, makes sense.

Any idea of an ETA when this will be available on npm?

Reply:
Will merge and publish tomorrow (Nov 15).